docs/book/usage/quilt.md

```diff
@@ -0,0 +1,87 @@
+# Quilt: Walrus Native Batch Store Tool
+
+Quilt is a batch storage feature, aiming to optimize the storage cost and efficiency of large
+numbers of small blobs. Prior to Quilt, storing small blobs (less than 10MB) in Walrus
```

Comment:
Nit: Can we use lower-case "quilt" except at the start of sentences? If we want to highlight it, I'd use italic font (`*quilt*`).